tracing/src/main/java/com/palantir/tracing/TraceState.java

```diff
@@ -79,12 +84,17 @@ String forUserAgent() {
         return forUserAgent;
     }
 
+    public Map<TraceLocal<?>, Object> getTraceLocals() {
+        return traceLocals;
+    }
+
     @Override
     public String toString() {
         return "TraceState{"
                 + "traceId='" + traceId + "', "
                 + "requestId='" + requestId + "', "
-                + "forUserAgent='" + forUserAgent
-                + "'}";
+                + "forUserAgent='" + forUserAgent + "', "
+                + "traceLocals=" + traceLocals
```

Comment:
We should not include the traceLocals map in toString because we have no guarantee that folks won't add tokens. Worth adding a test that traceLocal values aren't included in TraceState.toString
```suggestion
                + "traceLocals=" + traceLocals
```

Reply:
done